requirements/ci.txt

```diff
@@ -8,7 +8,7 @@ certifi==2021.10.8
     # via requests
 charset-normalizer==2.0.11
     # via requests
-codecov==2.1.12
+codecov==2.1.13
```

Comment:
I believe we can safely remove codecov from the requirements.

Reply:
I wanted to remove it too. I then noticed CI failing because it is run [here](https://github.com/open-craft/edx-enterprise/blob/9f8fb2ae9fbe4aac03b6b6cf6fa72e49046788f7/.github/workflows/ci.yml#L29-L31). I didn't want to introduce too many changes deviating from openedx's repo, if we can avoid it. The CI now seems to run fine. So, I am leaving it as it is for the moment.

Reply:
Now that I think about it, this is our nutmeg backport branch, which we might never reconcile with the upstream. So, removing it would actually be fine. Over thought that one.

Reply:
@tecoholic, ah, right, I didn't notice that this workflow was recently migrated to GH action - https://github.com/openedx/edx-enterprise/commit/f452ab66646d0cb57a7dd2fcbb06848f8007f42a. It's OK to keep it, then.